src/app/(Authentication)/SignUp/Address/index.tsx

```diff
@@ -46,7 +46,15 @@ export default function SignUpScreen() {
 
   const handleSubmit = () => {
     if (validateAddressInputs()) {
-      signUpUser(name, email, password, streetAddress, city, state, zipcode);
+      fullySignUpUser(
```

Comment:
Could you experiment with adding a `router.push()` call to advance the user to the AllCases screen after signup? I created an account and everything worked on the backend, but I was stuck on the address screen until I pressed back a few times. I think I just needed to clear the navigation stack to be in the correct place.